src/Commands/DocsCommand.ts

```diff
@@ -112,24 +116,23 @@ export class DocsCommand extends CommandBase {
      *
      * @param command Parsed out command
      * @param result Document to send
-     * @param currentPage Index representation of location the document
      * @param matches Language and term names
-     * @param originID ID for who sent the command message
      * @param message? If a message has already been sent, pass it here to overwrite instead
      *
      */
-    public static async sendDoc(command: CommandParser, result: Doc, currentPage: number, matches: string, originID: string, message?: any) {
+    public static async sendDoc(command: CommandParser, result: Doc, matches: string[], message?: any) {
 
         let messagePassed: boolean;
```

Comment:
I'd prefer if this variable `messagePassed` didn't need to exist. Right now its needed because if this method doesn't check if the trash emoji already has a collector from the `sendResults` function it will delete the message a second time and error.

Possible solution, attach the collect to message so it gets deleted at the same time? Not sure.